### PR TITLE
Upgrade @octokit/webhooks-types

### PR DESCRIPTION
**Pull request by aldenquimby**
**What's in this PR?**
- Upgrade `@octokit/webhooks` to the latest `@octokit/webhooks-types`

**Why**
- Existing types are outdated. E.g. see #2132

**Added feature flags**
- No

**Affected issues**  
- None

**How has this been tested?**  
- Since this is a TypeScript-only change, as long as the build passes it should be safe

**Whats Next?**
- nothing?

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -719,6 +719,11 @@
     graphql "^15.0.0"
     graphql-tag "^2.10.3"
```
Comment (aldenquimby) [inline yarn.lock:721]: I am unfamiliar with yarn, so if I messed something up here please let me know. This diff came from:
```
yarn remove @octokit/webhooks
yarn add -D @octokit/webhooks-types@latest
```

File: yarn.lock [hunk 2]
```diff
@@ -831,12 +831,10 @@
   dependencies:
     "@octokit/openapi-types" "^7.2.3"
```
Comment (aldenquimby) [inline yarn.lock:833]: I am unfamiliar with yarn, so if I messed something up here please let me know. This diff came from:
```
yarn remove @octokit/webhooks
yarn add -D @octokit/webhooks-types@latest
```

File: src/github/issue.ts [hunk 3]
```diff
@@ -28,7 +28,9 @@ export const issueWebhookHandler = async (context: WebhookContext<WebhookPayload
 	// TODO: need to create reusable function for unfurling
 	let linkifiedBody;
 	try {
-		linkifiedBody = await util.unfurl(issue.body);
+		if (issue.body !== null) {
+			linkifiedBody = await util.unfurl(issue.body);
+		}
```
Comment (aldenquimby) [inline src/github/issue.ts:33]: This is the only non-typescript change b/c the latest TS types allow `issue.body` to be null

**Review by gxueatlassian — APPROVED** [review]
Hi @aldenquimby thank you very much for your effort in this PR. I am happy to merge it in.

**Pull request merged**